service/src/main.rs

```diff
@@ -335,7 +335,7 @@ fn start_worker<E, T, D>(
 	let tx_hash = node_api.send_extrinsic(xthex, XtStatus::Finalized).unwrap();
 	println!("[<] Extrinsic got finalized. Hash: {:?}\n", tx_hash);
 
-	let last_synced_header = init_light_client(&node_api, enclave.as_ref());
+	let last_synced_header = init_light_client(&node_api, enclave.as_ref()).unwrap();
```

Comment:
If this fails, we shouldn't continue initializing the worker service - so the `unwrap()` here should be fine (everything else around it also uses `unwraps()`).

Reply:
Maybe we should make it clearer that we want to abort with an: `expect("Initializing LightClient failed. Aborting...")`